prow/config.yaml

```diff
@@ -1577,6 +1577,92 @@ postsubmits:
           - name: cache-ssd
             hostPath:
               path: /mnt/disks/ssd0
+      - name: ci-kubernetes-e2e-kubeadm-gce-upgrade-1-6-1-7
+        agent: kubernetes
+        spec:
+          containers:
+          - image: gcr.io/k8s-testimages/e2e-kubeadm:v20170920-20580b10
+            args:
+            - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+            - "--clean"
+            - "--git-cache=/root/.cache/git"
+            - "--timeout=320"
+            - "--upload=gs://kubernetes-jenkins/logs"
+            env:
+            - name: USER
+              value: prow
+            - name: GOOGLE_APPLICATION_CREDENTIALS
+              value: /etc/service-account/service-account.json
+            - name: JENKINS_GCE_SSH_PRIVATE_KEY_FILE
+              value: /etc/ssh-key-secret/ssh-private
+            - name: JENKINS_GCE_SSH_PUBLIC_KEY_FILE
+              value: /etc/ssh-key-secret/ssh-public
+            volumeMounts:
+            - name: service
+              mountPath: /etc/service-account
+              readOnly: true
+            - name: ssh
+              mountPath: /etc/ssh-key-secret
+              readOnly: true
+            - name: cache-ssd
+              mountPath: /root/.cache
+            ports:
+            - containerPort: 9999
+              hostPort: 9999
+          volumes:
+          - name: service
+            secret:
+              secretName: service-account
+          - name: ssh
+            secret:
+              secretName: ssh-key-secret
+              defaultMode: 0400
+          - name: cache-ssd
+            hostPath:
+              path: /mnt/disks/ssd0
+      - name: ci-kubernetes-e2e-kubeadm-gce-upgrade-1-7-1-8
```

Comment:
this should be chained after this job: https://github.com/kubernetes/test-infra/blob/master/prow/config.yaml#L16325